### PR TITLE
fixed several syntax issues in documentation

### PR DESCRIPTION
**Pull request by mvas**
I was looking through the docs and spotted several things that looked strange
that's a minor thing but would help the docs to look consistent

### CODE REVIEW

**Comment by blairconrad** [comment]
Thanks for this, @mvas! I'm disheartened (but not necessarily surprised) to see so many. I'll take a look as soon as time allows.

**Comment by blairconrad** [comment]
Time allowed. Thanks very much.

The only comment I have is that [in specifying-return-values.md](https://github.com/FakeItEasy/FakeItEasy/pull/749/files#diff-0d9d8fe75f59465688b3e8e52355e6d4R67) I'd probably shift `.ReturnsLazily` on line 67 to the left to line up with the `.SomeCall` on line 66, but your changes will substantially improve state of the docs, and the alignment is a very minor thing. If you don't have the time or interest, I'd be happy to submit a superseding PR, otherwise, I'd be thrilled to receive an amended commit.

@adamralph, @thomaslevesque, except my very minor note above, the changes all look good to me. I can see the pattern we (I) have when typing these things up!

**Comment by mvas** [comment]
I'd be able to update in 12 hrs, if that's too much, take the initiative

**Comment by thomaslevesque** [comment]
> I'd be able to update in 12 hrs, if that's too much, take the initiative

There's no rush, take your time :wink:

Thanks!

**Comment by mvas** [comment]
Done

**Comment by thomaslevesque** [comment]
> except my very minor note above, the changes all look good to me

To me too !

Merging now, thanks @mvas!

**Comment by blairconrad** [comment]
Indeed, @mvas. Thanks!
I am terrible with pictures of people. [This you?](https://twitter.com/vasyshchev)

File: docs/specifying-return-values.md [hunk 1]
```diff
@@ -63,8 +63,8 @@ than 4 parameters, the convenience methods won't work. Use the variant
 that takes an `IFakeObjectCall` instead:
 
 ```charp
-A.CallTo(objectCall => fakeShop.SomeCall(…)
-                               .ReturnsLazily(objectCall => calculateReturnFrom(objectCall));
+A.CallTo(() => fakeShop.SomeCall(…)
+                       .ReturnsLazily(objectCall => calculateReturnFrom(objectCall));
```
Comment (adamralph) [inline docs/specifying-return-values.md]: I think there's still a missing closing parenthesis here at the end of the first line?

Comment (blairconrad) [inline docs/specifying-return-values.md]: You are correct, sir!

Comment (blairconrad) [inline docs/specifying-return-values.md]: Raised #751.

Comment (mvas) [inline docs/specifying-return-values.md]: > I am terrible with pictures of people. This you?
Yes, @blairconrad 
does not seem to be terrible :)

Comment (mvas) [inline docs/specifying-return-values.md]: Sorry for missing that, @adamralph

Comment (blairconrad) [inline docs/specifying-return-values.md]: Don't be sorry. The three owners missed the mistakes in the original docs, and then 2/3 owners also missed that missing paren. You've made things much better. Thanks!

**Comment by blairconrad** [comment]
This issue has been fixed in [release 2.1.0](https://github.com/FakeItEasy/FakeItEasy/releases/tag/2.1.0).

Thanks for your help, @mvas. Look for your name in the release notes! :trophy:

**Pull request merged**